tests/structured-data/class-archive-post-test.php

```diff
@@ -318,7 +318,7 @@ public function test_archive_custom_post_type() {
 	 * @uses \Parsely::update_metadata_endpoint
 	 * @group metadata
 	 */
-	public function test_term_archive_custom_post_type() {
+	public function test_custom_term_archive() {
```

Comment:
```suggestion
	public function test_custom_taxonomy_term_archive() {
```

All _terms_ are custom ;-) - it's the _taxonomy_ that's custom that is the focus here.